src/javascripts/components/ChatInput.js

```diff
@@ -1,19 +1,23 @@
 import React from 'react'
 
 class ChatInput extends React.Component {
+
+  componentDidMount() {
+    this.inputText.focus();
+  }
+
   handleSubmit = (event) => {
     const { onSubmit } = this.props
     const text = this.inputText.value.trim()
-
-    if (this.inputText.value === "" || this.inputText.value === " ") {
-        console.log('You need text!')
-        this.inputText.focus()
+    if (text) {
+      onSubmit(text)
+      this.inputText.value = ""
+      this.inputText.focus()
+      event.preventDefault()
     } else {
-        onSubmit(text)
-        this.inputText.value = ""
-        this.inputText.focus()
+        event.preventDefault()
+        null
```

Comment:
what does this stray `null` do?

I think you can eliminate the entire `else` condition and just pull `event.preventDefault()` at the end, so it triggers in both cases without two lines of code